src/call.c

```diff
@@ -684,6 +687,18 @@ int call_alloc(struct call **callp, const struct config *cfg, struct list *lst,
 	if (msg && mbuf_get_left(msg->mb))
 		got_offer = true;
 
+	/* Check Call-Info header for ;answer-after=0 */
+	hdr = sip_msg_hdr(msg, SIP_HDR_CALL_INFO);
+	if (hdr) {
+		struct pl aprm;
+		if (0 == msg_param_decode(&hdr->val, "answer-after", &aprm)) {
+			if (0 == pl_strcasecmp(&aprm, "0")) {
+				info("call: answer-after=0\n");
+				call->should_answer = true;
+			}
```

Comment:
is it possible to have other values that "0" ?

what is the meaning of that value. if the semantics is a timer/timeout value should we perhaps
start an async timer in this case ?

Reply:
You're right. I guess, it was designed to use in conjuction with custom alert ringtone and probably short delay before call should be answered.
Some phones support another extra header: `Alert-Info` for this purpose.
There is example in [rfc 3261](https://tools.ietf.org/html/rfc3261#section-20.4)